Server/Pages/Shipment/RegisterShipment.razor

```diff
@@ -0,0 +1,31 @@
+@using Repositories.Entities
+@using Repositories.Interfaces
+
+@code {
+
+    [Inject]
+    public ISenderRepository SenderRepository { get; set; }
+    [Inject]
+    public IPackageRepository PackageRepository { get; set; }
+    [Inject]
+    public IPostMachineRepository PostMachineRepository { get; set; }
+    public Package Package { get; set; }
+    public IEnumerable<PostMachine> PostMachines { get; set; }
+
+    protected override async Task OnInitializedAsync()
+    {
+        Package = new Package();
+        await RetrievePostMachines();
+    }
+
+    public async Task RetrievePostMachines()
+    {
+        PostMachines = await PostMachineRepository.GetAsync();
+    }
+
+    public void CreateShipment()
```

Comment:
maybe CreatePackage would be more clear?

Reply:
also maybe this should be async also?

Reply:
I made it async, leaving create shipment for now because its after all RegisterShipment file, but could easily change it to create package if needed.